Drakhtar/GameObjects/Battalion.h

```diff
@@ -26,7 +27,7 @@ class Battalion : public Unit {
   virtual int getAttack() const;
   virtual int getMaxHealth() const;
 
-  virtual void loseHealth(int health);
+  virtual int loseHealth(int attack);
```

Comment:
`attack` isn't pretty self-explanatory, and almost looks like a typo or bad copypasta, `damage` would make more sense.

```suggestion
  virtual int loseHealth(int damage);
```

Reply:
Although this could also be renamed to `dealDamage(int damage);`